tests/helpers/bg_server_sleep.tcl

```diff
@@ -4,7 +4,7 @@ source tests/support/util.tcl
 proc bg_server_sleep {host port sec} {
     set r [valkey $host $port 0]
     $r client setname SLEEP_HANDLER
-    $r debug sleep $sec
+    catch {$r debug sleep $sec}
```

Comment:
We generally don't put the server to sleep like this, because it becomes completely unresponsive. We typically use `pause_process` and `resume_process` instead. I'm going to run the tests and merge it if it works to make the tests green, but we should follow up and try to avoid the usage of debug sleep here.